### PR TITLE
Refactoring/khwup 120/refactor strings constants in lit cli project

### PR DESCRIPTION
**Pull request by olga-biriukova**
Change string constants usage
Task khwup-120-refactor strings constants...

### CODE REVIEW

File: lit/command/CommitCommand.py [hunk 1]
```diff
@@ -1,59 +1,57 @@
...
 
         myzip_hash = self.get_file_hash(myzip.filename)
+        message = StringsHolder.Commands.Commit.Arguments.MSG_NAME.value
         commit = {
-            SettingsManager.get_var_value('COMMIT_USER'): 'WIP',
-            SettingsManager.get_var_value('COMMIT_LONG_HASH'): myzip_hash,
-            SettingsManager.get_var_value('COMMIT_SHORT_HASH'): myzip_hash[:10],
-            SettingsManager.get_var_value('COMMIT_DATETIME'): str(datetime.utcnow()),
-            SettingsManager.get_var_value('COMMIT_COMMENT'): args['message'],
+            StringsHolder.CommitSettings.USER: 'WIP',
```
Comment (pavlobielousov) [inline lit/command/CommitCommand.py]: This way of storing strings is not suitable in this case. You should be able to change VCS user's name, but in this case the user's name is hardcoded in python source file and cannot be changed except modifying the sources.

File: lit/strings_holder.py [hunk 2]
```diff
@@ -2,68 +2,87 @@
...
+        FILE_NAME = 'commits_log.json'
+        INIT_CONTENT = '{"commits":[]}'
+        KEY = "commits"
+
+    class CommitSettings(Enum):
+        DIR_PATH = '.lit/commits/'
+        DIR_NAME = 'commits'
+        ZIP_FILE_NAME = '.lit/commits/hash'
+        ZIP_EXTENSION = '.zip'
+        USER = 'user'
```
Comment (pavlobielousov) [inline lit/strings_holder.py]: User name should be changeable externally from CLI. Here we can add some method to get/set user name and store it on disk in JSON/SQLite/etc.

**Review by pavlobielousov — COMMENTED** [review]
The way of storing variable settings should be implemented, instead of hardcoding the settings.

File: lit/strings_holder.py (hunk 2 above)
Comment (maxkrivich) [reply lit/strings_holder.py]: Probably best way to store settings it's using configuration files. Check configparser module.

Comment (olga-biriukova) [reply lit/strings_holder.py]: This settings are constants, they will not be changed. Configparse good idea for implementing storing user information. In other task.

File: lit/command/CommitCommand.py (hunk 1 above)
Comment (olga-biriukova) [reply lit/command/CommitCommand.py]: We can solve it in another branch...

**Pull request closed without merging**